static/app/views/issueList/savedIssueSearches.tsx

```diff
@@ -165,4 +226,26 @@ const SavedSearchItemDescription = styled('div')`
   text-overflow: ellipsis;
 `;
 
+const OverflowMenu = styled(DropdownMenuControl)`
+  position: absolute;
+  top: 12px;
+  right: ${space(1)};
+`;
+
+const OverflowMenuTrigger = styled(Button)<{isActive?: boolean}>`
+  border: 0;
+  height: 30px;
+  width: 30px;
+  background: ${p => (p.isActive ? p.theme.hover : 'none')};
+  color: ${p => p.theme.textColor};
+  box-shadow: none;
+  cursor: pointer;
+
+  &:hover,
+  &:focus {
+    background: ${p => p.theme.hover};
+    color: ${p => p.theme.textColor};
```

Comment:
I wouldn't worry about hover state color change. I would prefer to just not have this styled component and just use a default button with `borderless` and the appropriate `size`

Reply:
Good note, I like that much better: https://github.com/getsentry/sentry/pull/40898/commits/e8feb121d52eff030d7e99d00b1796d76e4d88e8